sdk/storage/azure-storage-blobs/test/ut/append_blob_client_test.cpp

```diff
@@ -7,103 +7,101 @@
+    EXPECT_EQ(blobClient.GetTags().Value, createOptions.Tags);
 
     auto blockContent
         = Azure::Core::IO::MemoryBodyStream(m_blobContent.data(), m_blobContent.size());
-    auto appendResponse = appendBlobClient.AppendBlock(blockContent);
-    properties = appendBlobClient.GetProperties().Value;
+    auto appendResponse = blobClient.AppendBlock(blockContent);
+    properties = blobClient.GetProperties().Value;
     EXPECT_EQ(properties.CommittedBlockCount.Value(), 1);
     EXPECT_EQ(properties.BlobSize, static_cast<int64_t>(m_blobContent.size()));
 
     Azure::Storage::Blobs::AppendBlockOptions options;
     options.AccessConditions.IfAppendPositionEqual = 1_MB;
     blockContent = Azure::Core::IO::MemoryBodyStream(m_blobContent.data(), m_blobContent.size());
-    EXPECT_THROW(appendBlobClient.AppendBlock(blockContent, options), StorageException);
+    EXPECT_THROW(blobClient.AppendBlock(blockContent, options), StorageException);
     options.AccessConditions.IfAppendPositionEqual = properties.BlobSize;
     blockContent = Azure::Core::IO::MemoryBodyStream(m_blobContent.data(), m_blobContent.size());
-    appendBlobClient.AppendBlock(blockContent, options);
+    blobClient.AppendBlock(blockContent, options);
 
-    properties = appendBlobClient.GetProperties().Value;
+    properties = blobClient.GetProperties().Value;
     options = Azure::Storage::Blobs::AppendBlockOptions();
     options.AccessConditions.IfMaxSizeLessThanOrEqual
         = properties.BlobSize + m_blobContent.size() - 1;
     blockContent = Azure::Core::IO::MemoryBodyStream(m_blobContent.data(), m_blobContent.size());
-    EXPECT_THROW(appendBlobClient.AppendBlock(blockContent, options), StorageException);
+    EXPECT_THROW(blobClient.AppendBlock(blockContent, options), StorageException);
     options.AccessConditions.IfMaxSizeLessThanOrEqual = properties.BlobSize + m_blobContent.size();
     blockContent = Azure::Core::IO::MemoryBodyStream(m_blobContent.data(), m_blobContent.size());
-    appendBlobClient.AppendBlock(blockContent, options);
+    blobClient.AppendBlock(blockContent, options);
 
-    properties = appendBlobClient.GetProperties().Value;
+    properties = blobClient.GetProperties().Value;
     int64_t originalLength = properties.BlobSize;
-    appendBlobClient.AppendBlockFromUri(client.GetUrl() + GetSas());
-    properties = appendBlobClient.GetProperties().Value;
-    EXPECT_EQ(properties.BlobSize, static_cast<int64_t>(originalLength + m_blobContent.size()));
+    blobClient.AppendBlockFromUri(blobClient.GetUrl() + GetSas());
+    properties = blobClient.GetProperties().Value;
+    EXPECT_EQ(properties.BlobSize, 2 * originalLength);
 
-    auto deleteResponse = appendBlobClient.Delete();
+    auto deleteResponse = blobClient.Delete();
     EXPECT_TRUE(deleteResponse.Value.Deleted);
-    EXPECT_THROW(appendBlobClient.Delete(), StorageException);
-  }
-
-  TEST_F(AppendBlobClientTest, CreateWithTags)
```

Comment:
why are you removing this test ?

Reply:
This is already covered in test case `AppendBlobClientTest.CreateAppendDelete`